### ISSUE TITLE
NTR, pulmo: Centrilobular Emphysema

### ISSUE THREAD

**Issue by GrieseAG** [obophenotype/human-phenotype-ontology#5578]
**Preferred term label:**
Centrilobular Emphysema

**Synonyms**
centriacinar emphysema

**Definition (free text, please give PubMed ID)**
In the pathology, they are characterized by destroyed centrilobular alveolar walls and enlargement of respiratory bronchioles and associated alveoli (42,43). This is the commonest form of emphysema in cigarette smokers. CT findings are centrilobular areas of decreased attenuation, usually without visible walls, of nonuniform distribution and predominantly located in upper lung zones

PMID: 18195376, PMID: 4014865, PMID: 7976869

**Parent term (use hpo.jax.org/app)**
Emphysema HP:0002097

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
LMU crapp, @mgriese, @katrinknoflach

**Comment by pnrobinson** [comment]
added term
